tracer/src/Datadog.Trace/Ci/Coverage/Attributes/AvoidCoverageAttribute.cs

```diff
@@ -0,0 +1,20 @@
+// <copyright file="AvoidCoverageAttribute.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using Datadog.Trace.Ci.Coverage.Attributes;
+
+[assembly: AvoidCoverage]
```

Comment:
Could this instead go in the `AssemblyInfo.cs` file so our assembly attributes are declared in one place?

Reply:
done in [d0d8115](https://github.com/DataDog/dd-trace-dotnet/pull/2739/commits/d0d8115612e93d65e6a09402f69f3443e1a35bfb)